packages/nvidia-container-toolkit/nvidia-container-toolkit.spec

```diff
@@ -30,15 +30,15 @@ Requires: %{_cross_os}shimpei
 
 %build
 %cross_go_configure %{goimport}
-go build -buildmode=pie -ldflags="${GOLDFLAGS}" -o nvidia-container-toolkit ./cmd/nvidia-container-toolkit
+go build -buildmode=pie -ldflags="${GOLDFLAGS}" -o nvidia-container-runtime-hook ./cmd/nvidia-container-runtime-hook
 
 %install
 install -d %{buildroot}%{_cross_bindir}
 install -d %{buildroot}%{_cross_tmpfilesdir}
 install -d %{buildroot}%{_cross_templatedir}
 install -d %{buildroot}%{_cross_datadir}/nvidia-container-toolkit
 install -d %{buildroot}%{_cross_factorydir}/etc/nvidia-container-runtime
-install -p -m 0755 nvidia-container-toolkit %{buildroot}%{_cross_bindir}/nvidia-container-runtime-hook
+install -p -m 0755 nvidia-container-runtime-hook %{buildroot}%{_cross_bindir}/
```

Comment:
extremely minor nit: I try to avoid trailing slashes for directories like this since, if done habitually, it tends to lead to double slashes (`//`) in path names in build output, which is harmless but annoying.

Not worth fixing here, it's mostly about asking the question of whether a slash is needed after a given macro when you're writing spec files.